lib/protocol/protocol.js

```diff
@@ -8,9 +8,9 @@ import { renameKey } from '../basedriver/helpers';
 import B from 'bluebird';
 import BaseDriver from '../basedriver/driver';
 
-
 const mjsonwpLog = logger.getLogger('MJSONWP');
 const w3cLog = logger.getLogger('W3C');
+const genericProtocolLog = logger.getLogger('GENERIC');
```

Comment:
should this be `GENERIC` or `UNKNOWN_PROTOCOL`?

Reply:
I prefer GENERIC, because we include data for both protocols into the response if the protocol is neither W3C nor JSONWP